modules/oe_content_event/config/install/field.field.node.oe_event.field_registration_url.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_registration_url
+    - node.type.oe_event
+  module:
+    - link
+id: node.oe_event.field_registration_url
+field_name: field_registration_url
+entity_type: node
+bundle: oe_event
+label: 'Registration URL'
+description: ''
+required: false
```

Comment:
should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Registration fields group filled in.